chicommons/maps
# Resolves 'module not found' issue

Updates docker image to use node 20. Bumps React to 16.14 and pins other package versions to resolve dependency tree issues.

---
## Testing
- Remove the `node_modules` folder (cd into `client` and run `sudo rm -rf node_modules`)
- Remove `yarn.lock`
- From root of project run `docker-compose up`
- Confirm that frontend app renders without error

*note*: if you still have a frontend error, you may need to remove the cached docker images with `docker system prune -a` and then re-run `docker-compose up`
